public_html/wp-content/mu-plugins/blocks/assets/src/speakers/speakers-select.js

```diff
@@ -61,6 +67,7 @@ class SpeakersSelect extends Component {
 
 	static optionLabel( optionData ) {
 		const { type } = optionData;
+		// would it be better to include label and count in ^, and then name the return value something like `labelMarkup` ?
```

Comment:
We could. I think I split out the image and the label into separate methods to keep things shorter, but it might make more sense for them to be combined...